src/main/java/net/greenfieldmc/core/advancedbuild/handlers/MultipleFacingInteraction.java

```diff
@@ -31,6 +32,7 @@ public TextComponent getInteractionUsage() {
     @Override
     public void onRightClickBlock(PlayerInteractEvent event) {
         if (!event.getPlayer().isSneaking()) return;
+        if (event.getPlayer().getInventory().getItemInMainHand().getType() != Material.AIR) return;
```

Comment:
If you want this interaction to work ONLY when you are holding an air block, move this check to the predicate in the constructor, see first comment in this file.